Makefile

```diff
@@ -31,3 +31,7 @@ test:
 verify:
 	goimports -d .
 	golint ./...
+
+protos/gen_mock:
+	go generate ./protos/**/*
+	goimports -w ./protos/**/mock.go
```

Comment:
Would be great if these were proper deps for the necessary binaries so that they would always be up to date. Doesn't have to be now, I really want to see this PR done with :D. I can open up an issue when this PR is merged though